projects/applicationinsights-angularplugin-js/src/lib/applicationinsights-angularplugin-error.service.ts

```diff
@@ -8,6 +8,8 @@ import { IErrorService } from './IErrorService';
 })
 export class ApplicationinsightsAngularpluginErrorService implements IErrorService {
     public static instance: ApplicationinsightsAngularpluginErrorService = null;
+    // What's the purpose of making this the only instance for the class?
```

Comment:
No, this is so that the Error Service knows which analytics pluin instance it should use to send any reported error.

Without this there is no "link" for the error handler to be able to "report" the errors, (if) they used the snippet then use there would be the `appInsights` global, however, that doesn't work (is not available) when using npm. And when using Angular most people (I would guess something like 99.9999999% use `npm` for their own bundles.